src/lib/integration_test_cloud_engine/mina_automation.ml

```diff
@@ -340,8 +340,8 @@ module Network_config = struct
         ; aws_route53_zone_id
         ; cpu_request = 6
         ; mem_request = "12Gi"
-        ; worker_cpu_request = 4
-        ; worker_mem_request = "6Gi"
+        ; worker_cpu_request = 8
+        ; worker_mem_request = "12Gi"
```

Comment:
Did these make any difference? zkApp tests which generally require more snark cpu were running fine. cc @stevenplatt 
Do you have links to CI runs without this change?

Reply:
It's hard to say but I think they did.  Ever since making this change, i've never gotten the timeouts on waiting for the proof to be emitted.

Reply:
Can we confirm before doing that since this impacts resource allocation/cluster configuration/costs. I remember Steven checking the resource usage and said it was within the previous limits. Also, these are payments and so shouldn't be needed more than what we've always had

Reply:
OK let me do a branch with this reverted for comparison